korio/src/commonMain/kotlin/com/soywiz/korio/async/withOptTimeout.kt

```diff
@@ -2,7 +2,7 @@ package com.soywiz.korio.async
 
 import kotlinx.coroutines.*
 
-suspend fun <T> withOptTimeout(ms: Int?, name: String = "timeout", callback: suspend () -> T): T {
+suspend fun <T> withOptTimeout(ms: Long?, name: String = "timeout", callback: suspend () -> T): T {
```

Comment:
I guess we should change `ms: Long` to `time: TimeSpan`. Also double boxing in JS (internal js Long box + Long object for nullable).